apps/gpstrek/app.js

```diff
@@ -318,16 +318,24 @@ function triangle (x, y, width, height){
   ];
 }
 
+function onSwipe(dir){
+  if (dir < 0) {
+    nextScreen();
+  } else if (dir > 0) {
+    switchMenu();
+  } else {
+    nextScreen();
+  }
+}
+
 function setButtons(){
-  Bangle.setUI("leftright", (dir)=>{
-    if (dir < 0) {
-      nextScreen();
-    } else if (dir > 0) {
-      switchMenu();
-    } else {
-      nextScreen();
-    }
-  });
+  let options = {
+    mode: "custom",
+    swipe: onSwipe,
+    btn: nextScreen,
+    touch: nextScreen
+  };
+  Bangle.setUI(options);
```

Comment:
I just wanted to check, does this break Bangle.js 1? Or does it still work on both?

Reply:
Forget that - it never supported the 1 anyway ;)